adapters/ix/ix.go

```diff
@@ -273,6 +283,23 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 		}
 	}
 
+	if bidResponse.Ext != nil {
+		var bidRespExt ixRespExt
+		if err := json.Unmarshal(bidResponse.Ext, &bidRespExt); err == nil && bidRespExt.AuctionConfig != nil {
```

Comment:
should not ignore `unMarshal` error. This could be a potential bad bidder server issue

Reply:
added error check